fmriprep/workflows/util.py

```diff
@@ -180,11 +191,14 @@ def init_bbreg_wf(bold2t1w_dof, report, reregister=True, name='bbreg_wf'):
         :simple_form: yes
 
         from fmriprep.workflows.util import init_bbreg_wf
-        wf = init_bbreg_wf(bold2t1w_dof=9, report=False)
+        wf = init_bbreg_wf(use_bbr=True, bold2t1w_dof=9, report=False, omp_nthreads=1)
```

Comment:
Using `use_bbr=True` here to keep the workflow cleaner; the selection-related nodes don't seem informative to me.